common/src/main/scala/tracking/NotificationReportRepository.scala

```
@@ -35,30 +36,48 @@ class NotificationReportRepository(client: AsyncDynamo, tableName: String)
       return Future.successful(Left(RepositoryError("Date range too big to query")))
     }
 
-    def maybeLastKey(result: QueryResult): Option[util.Map[String, AttributeValue]] = Option(result.getLastEvaluatedKey).flatMap(lastKey => if (lastKey.isEmpty) None else Some(lastKey))
+    def maybeStartKey(result: QueryResult): Option[util.Map[String, AttributeValue]] = Option(result.getLastEvaluatedKey).flatMap(lastKey => if (lastKey.isEmpty) None else Some(lastKey))
 
-    def reportsFromResult(result: QueryResult): List[NotificationReport] = result.getItems.asScala.toList.flatMap { item =>
-      fromAttributeMap[NotificationReport](item.asScala.toMap).asOpt
```

`.asOpt` is where the parsing errors get swallowed